lab_1_classify_by_unigrams/start.py

```diff
@@ -5,6 +5,7 @@
 
 
 def main() -> None:
+    from main import create_language_profile
```

Comment:
all imports are located at the top of any python file

Reply:
done